src/main.rs

```diff
@@ -18,7 +20,14 @@ async fn main() -> Result<(), Error> {
         if !args.keep_single_end {
             runs.iter_mut().for_each(|run| run.clean_single_end());
         }
-        println!("{}", serde_json::to_string_pretty(&runs).unwrap());
+        if std::mem::discriminant(&output_fmt) == std::mem::discriminant(&OutputFormat::Csv) {
```

Comment:
Checkout lines 105-111 of lib.rs for a suggestion on how to handle the error here. You could. add exit(1) at the end, as this is bad error and we don't really expect it to happen.